### PR TITLE
deploy fix test

### PR DESCRIPTION
**Pull request by mshaban-msft**
this is just a draft to showcase proposed changes

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json [hunk 1]
```diff
@@ -456,6 +456,12 @@
           "application/json"
         ],
         "responses": {
+          "200": {
+            "description": "Deploy job state.",
+            "schema": {
+              "$ref": "#/definitions/JobState"
+            }
+          },
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json:464]: I'm confused. An LRO could return either a 200 with the completed state, or a location for updating an LRO object state? What would a method look like in this case for statically-typed languages like C# and Java? How would customers know what they will get, or is the idea that an operation object e.g., `Operation<JobState>` in .NET, will just come back in a completed state?

/cc @johanste

Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json:464]: @johanste, @JeffreyRichter is this a proper fix based on the guidelines for LRO (linked below)? I thought maybe these two headings were mutually exclusive, but perhaps that's not the case?

* [Resource-based LROs](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#resource-based-long-running-operations)
* [Long-running operations with status monitor](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations-with-status-monitor)

The guidelines right above that even say,

> There are two basic patterns that can be used for long-running operations:
>
> * Resource-based long-running operations (RELO)
> * Long-running operations with status monitor

Should an endpoint really be supporting both?

Comment (JeffreyRichter) [reply specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json:464]: An endpoint should not support both; they are mutually-exclusive. @mikekistler can help with what the swagger should look like.

Comment (mikekistler) [reply specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json:464]: @heaths Our API Guidelines have been clarified over the last 6 months.  In the updated guidelines, there is a clear distinction between resource-based LROs and status-monitor LROs. They are mutually exclusive.  In particular, a resource-based LRO _never_ returns a `202 Accepted` -- it returns a `200` and/or `201`.  And a status-monitor LRO _only_ returns `202` as the success status code. No other success status codes are allowed [[link to Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=recently%20initiated%20operations.-,%E2%9A%A0%EF%B8%8F,even%20if%20processing%20was%20completed%20before%20the%20initiating%20request%20returns.,-%E2%9B%94)].

So if this operation already returns `202 Accepted`, its die is cast.  It is a status-monitor LRO and must stay way.

As for what the swagger should look like, I think you just can't make the change you are proposing. If you were doing that to solve some particular problem, you will need to find another way to solve it.

Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json:464]: @mshaban-msft, as the PR author, please see @mikekistler's response.

**Pull request closed without merging**